### PR TITLE
chore: i18n files url add version query parameter

### PR DESCRIPTION
**Pull request by tyyzqmf**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

## Summary

(describe what this merge request does)

## Implementation highlights

(describe how the merge request does for feature changes, share the RFC link if it has)

## Test checklist

- [ ] add new test cases
- [x] all code changes are covered by unit tests
- [ ] end-to-end tests
  - [ ] deploy web console with CloudFront + S3 + API gateway
  - [ ] deploy web console within VPC
  - [ ] deploy ingestion server
    - [ ] with MSK sink
    - [ ] with KDS sink
    - [ ] with S3 sink
  - [ ] deploy data processing
  - [ ] deploy data modeling
    - [ ] new Redshift Serverless
    - [ ] provisioned Redshift
    - [ ] Athena
  - [ ] deploy with reporting
  - [ ] streaming ingestion
    - [ ] with Redshift Serverless
    - [ ] with provisioned Redshift

## Is it a breaking change

- [ ] add parameters without default value in stack
- [ ] introduce new service permission in stack
- [ ] introduce new top level stack module

## Miscellaneous

- [ ] introduce new symbol link source file(s) to be shared among infra code, web console frontend, and web console backend

### CODE REVIEW

**Review by chenhaiyun — APPROVED** [review]

File: frontend/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_SOLUTION_VERSION=
```
Comment (zxkane) [inline frontend/.env:1]: Is it a mandatory file?  Could we only set the environment variable `REACT_APP_SOLUTION_VERSION` when building frontend code?

File: .projenrc.js [hunk 2]
```diff
@@ -211,6 +211,7 @@ const devDepsForFrontendProject = [
   'html-minifier@^4.0.0',
   'prettier@^2.7.1',
   'typescript@^4.8.4',
+  'env-cmd@^10.1.0',
```
Comment (zxkane) [inline .projenrc.js:214]: Do we still need this tool?

File: src/cloudfront-control-plane-stack.ts [hunk 3]
```diff
@@ -222,6 +222,7 @@ export class CloudFrontControlPlaneStack extends Stack {
           'mkdir $APP_PATH',
           'tar --exclude=\'node_modules\' --exclude=\'cdk.out\' --exclude=\'build\' -cf - . | (cd $APP_PATH && tar -xf -)',
           'cd $APP_PATH',
+          'export REACT_APP_SOLUTION_VERSION=$REACT_APP_SOLUTION_VERSION',
```
Comment (zxkane) [inline src/cloudfront-control-plane-stack.ts:225]: Is it useless once you have already assigned the environment?

**Review by zxkane — CHANGES_REQUESTED** [review]
- [x] rebase and resolve locally

**Review by zxkane — APPROVED** [review]

**Pull request merged**